tests/c/trace_too_long.c

```diff
@@ -0,0 +1,46 @@
+// # This test can be very slow, so ignore it except where we know it'll run
+// # fast enough.
+// ignore-if: test "$YKB_TRACER" != "hwt"
+// Run-time:
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     ...
+//     jit-state: stop-tracing
+//     jit-state: trace-compilation-aborted: Trace too long
+
```

Comment:
What is the intention of this test? To check that long traces are aborted I think? It's conventional to add a short comment saying what the test is checking.

Reply:
Lengthened description in 3fafef6.